diff --git a/dist/cjs/instrumentation/env.js b/dist/cjs/instrumentation/env.js
index ec438baee8e067449ac9e65cae607b98d192b657..5134e96460a00a9335fd363070cf710ec3da534e 100644
--- a/dist/cjs/instrumentation/env.js
+++ b/dist/cjs/instrumentation/env.js
@@ -16,7 +16,7 @@ const isDurableObject = (item) => {
     return !!(item === null || item === void 0 ? void 0 : item.idFromName);
 };
 const isFetcher = (item) => {
-    return item instanceof Fetcher;
+    return item.hasOwnProperty('fetch');
 };
 const instrumentEnv = (env) => {
     const envHandler = {
diff --git a/dist/esm/instrumentation/env.js b/dist/esm/instrumentation/env.js
index 8f91f54bc0d75d709332903f6ef61267e707ac1e..c3c9e8427736969eb85b3310ca282edf9a7879f4 100644
--- a/dist/esm/instrumentation/env.js
+++ b/dist/esm/instrumentation/env.js
@@ -13,7 +13,7 @@ const isDurableObject = (item) => {
     return !!item?.idFromName;
 };
 const isFetcher = (item) => {
-    return item instanceof Fetcher;
+    return item.hasOwnProperty('fetch');
 };
 const instrumentEnv = (env) => {
     const envHandler = {
